x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/views/paging_view.tsx

```diff
@@ -10,14 +10,14 @@ import React from 'react';
 import { EuiPagination } from '@elastic/eui';
 
 interface Props {
-  current: number;
+  current?: number;
   totalPages: number;
   onChange(pageNumber: number): void;
   'aria-label': string;
 }
 
 export const PagingView: React.FC<Props> = ({
-  current,
+  current = 1,
```

Comment:
Search UI says that this prop is optional but that might be incorrect, added to issue to investigate: https://github.com/elastic/enterprise-search-team/issues/2226